munch-match/.env-template

```
@@ -1 +1,2 @@
-REACT_APP_GOOGLE_API_KEY = 
\ No newline at end of file
+REACT_APP_GOOGLE_API_KEY = AIzaSyBTDgz5uQ2c_lxNFzUGqudqF7y6e31JCzw
```

Might just want to remove the Google API key value here as users are meant to generate their own. Not sure how the other API key works but if you're good with leaving it in, then that should be fine
```suggestion
REACT_APP_GOOGLE_API_KEY =
```

---

Actually probably safe to remove the other API key from the template too just for security